### PR TITLE
feat(buttons): Add new design styles and improvements

### PR DESCRIPTION
**Pull request by theOrlin**

### CODE REVIEW

File: scss/button/_layout.scss [hunk 1]
```diff
@@ -118,9 +118,21 @@
     }
 
     //DropDown Button
-    kendo-dropdownbutton.k-state-focused,
-    kendo-dropdownbutton:focus {
-        outline: 0;
+    .k-dropdownbutton {
```
Comment (joneff) [inline scss/button/_layout.scss]: If we have `k-split-button`, then this should be `k-dropdown-button`, or vice versa -- `k-splitbutton` and `k-dropdownbutton`.

The rest is fine.

Comment (joneff) [reply scss/button/_layout.scss]: @gyoshev @nlazarov  any thoughts on this?

**Review by joneff — CHANGES_REQUESTED** [review]

File: scss/button/_theme.scss [hunk 2]
```diff
@@ -1,9 +1,8 @@
 // TODO: button variables
-$button-focused-box-shadow: 0 0 0 2px darken($widget-bg, 6%) !default;
-$button-active-box-shadow: inset 0 3px 5px darken($widget-bg, 9%) !default;
+$button-focused-box-shadow: 0 3px 4px 0 rgba(0, 0, 0, .06) !default;
+$button-active-box-shadow: inset 0 2px 2px 0 rgba(0, 0, 0, .06) !default;
 
-$primary-focused-box-shadow: 0 0 0 2px lighten($primary-button-bg, 23%) !default;
-$primary-active-box-shadow: inset 0 3px 5px desaturate($primary-button-bg, 3%) !default;
+$primary-focused-box-shadow: 0 3px 4px 0 rgba(255, 99, 88, .06) !default;
```
Comment (gyoshev) [inline scss/button/_theme.scss]: How is this color calculated? seems related to `$accent`.

File: scss/button/_layout.scss (hunk 1 above)
Comment (gyoshev) [reply scss/button/_layout.scss]: Seems to be the same question that we had for the `button-group` and `buttongroup`, no? telerik/kendo-angular-buttons#45

File: scss/button/_theme.scss (hunk 2 above)
Comment (theOrlin) [reply scss/button/_theme.scss]: It is exactly the $accent color. I was unaware that I could use a hex variable inside an RGBA function in SASS. Corrected now.

File: scss/button/_layout.scss (hunk 1 above)
Comment (theOrlin) [reply scss/button/_layout.scss]: Changed the class name to k-dropdown-button.

**Review by joneff — APPROVED** [review]

**Pull request merged**